### ISSUE TITLE
Main search bar does not have a label

### ISSUE THREAD

**Issue by piet0024** [gctools-outilsgc/gccollab#111]
The main search region has an invisible label but it is not associated to the input.

@CoolColin

**Comment by markwooff** [comment]
Label fixed to point to the search input field.
